### PR TITLE
Add Marlin, Gerrit, Nilay and Eeshan to AUTHORS. Welcome to SymEngine!

### PR DESCRIPTION
**Pull request by isuruf**
@Marlin-Na, @Wrzlprmft, @npochhi, @eeshan9815, thanks for your contributions. I've added you to the authors list. Let me know if you want to change your name or email address.

### CODE REVIEW

**Comment by Marlin-Na** [comment]
@isuruf Thanks! I would like to change my name to "Jialin Ma"

**Comment by Wrzlprmft** [comment]
The e-mail address for me (Gerrit Ansmann) is wrong. Please use none or `wrzlrpmft@users.noreply.github.com`. If anybody really needs to contact me and GitHub doesn’t suffice, they can find out with a bit of research.

File: AUTHORS [hunk 1]
```diff
@@ -52,4 +52,8 @@ Kv Manohar <kvmanohar22@gmail.com>
 Kanchana Ruwanpathirana <kanchana.ruwanpathirana@gmail.com>
 Melanka Saroad <saroadmelanka@yahoo.com>
 Dirk Reusch <lists@kybdr.de>
+Jialin Ma <marlin-@gmx.cn>
+Gerrit Ansmann <wrzlrpmft@users.noreply.github.com>
```
Comment (isuruf) [inline AUTHORS]: @Wrzlprmft, fixed.

**Comment by npochhi** [comment]
@isuruf Thanks! 😄

**Comment by isuruf** [comment]
Ping @eeshan9815

**Comment by eeshan9815** [comment]
Thanks!
My name is "Eeshan Gupta" and email ID is "eeshan9815@gmail.com"

**Pull request merged**